core/shared/src/main/scala/zio/stm/ZSTM.scala

```diff
@@ -1617,10 +1647,13 @@ object ZSTM {
   private[zio] def succeedNow[A](a: A): USTM[A] = SucceedNow(a)
 
   private[stm] object internal {
-    val DefaultJournalSize = 4
-    val MaxRetries         = 10
-    val YieldOpCount       = 2048
+    final val DefaultJournalSize   = 3
```

Comment:
Curious why 4 -> 3?

Reply:
Previously we used a java.util.HashMap and passed `4` in the constructor. Because the constructors of Scala's HashMap are different for 2.12 and 2.13+, I had to use a version-specific method that uses the intended number of elements than the actual capacity. So 3 / 0.75d = 4

Reply:
Added a comment with an explanation on why 3